.github/workflows/build.yml

```diff
@@ -16,41 +16,21 @@ jobs:
       fail-fast: false
       matrix:
         ruby:
+          - 3.1
+          - 3.0
           - 2.7
           - 2.6
-          - 2.5
-          - 2.4
-          - 2.3
-          - 2.2
         gemfile:
-          - Gemfile-5-0
-          - Gemfile-5-1
-          - Gemfile-5-2
-          - Gemfile-6-0
```

Comment:
since you remove rails 6.0 and ruby 2.5. can you bump the minimums in the `uob-payroll.gemspec` file ?